lib/rewards/server/subscribers/rewards-subscriber.js

```diff
@@ -0,0 +1,135 @@
+const { sc, Logger } = require('@reldens/utils');
+
+class RewardsSubscriber
```

Comment:
same as above, this would also be a handler (but don't worry for now I did the same for other subscribers).

on the other hand on what you need to focus is to keep the classes with a single responsability as much as possible, I checking now the multiple methods this one has

Reply:
(Y)